packages/core/src/cloud/config.ts

```diff
@@ -16,7 +16,7 @@ function getTransformationConfig(
 ): PlatformApi.ConfiguredTransformationItem {
   if (transformation instanceof SimpleNodeParser) {
     return {
-      configurableTransformationType: "SENTENCE_AWARE_NODE_PARSER",
+      configurable_transformation_type: "SENTENCE_AWARE_NODE_PARSER",
```

Comment:
didn't fern have a feature to support camel case for TS?

Reply:
I don't use fern in TS side for the latest code, but it's okay to use any type generation in SDK side